### ISSUE TITLE
Color of dots

### ISSUE THREAD

**Issue by ebgoldstein** [ebgoldstein/IrmaTweets#2]
Could be changed combined scorein stead of just image score?

**Comment by elk2klein** [comment]
Agreed, combined score might be more useful, or a radio button option to select which score?

**Comment by ebgoldstein** [comment]
Actually this is tricky for just combined score — i like the idea of radio button for the 4 scores.

**Comment by ebgoldstein** [comment]
in the `ui`, need a select input:
`selectInput("colorvariable", "Color based on:", c("Image Score"="imageScore", "Text Score"="textScore", "User Score"="userScore", "GIS Score"="gisScore")),`

and then make this work with `fillColor` and the legend observer

**Comment by ebgoldstein** [comment]
for some reason i just can't make this work...
